docs/README.md

```diff
@@ -0,0 +1,47 @@
+## CPMA capabilities
+
+This documents list CPMA capabilities.
+
+---
+
+### Report
+
+Report consist of 2 parts:
+
+1. Pre-migration analytics that goes under "cluster" in json. It contains general information about: 
+  * Nodes - name, сpu usage, memory capacity, consumed memory, number of running pods, pod capacity, information that
+  indicates if node is master.
+  * Namespaces - name, latest change timestamp, container count, total cpu/memory usage.
+    * Pods - name
+    * Route - name, spec
+    * Daemonsets - name, latest change timestamp
+    * Deployments - name, latest change timestampg
+    * Quotas - name, spec, selectors(labels, annotations)
+  * Storage classes - name, provisioner
+  * RBAC - information about users, groups, roles, cluster roles, cluster role bindings, security context constraints
+  * Persistent volumes - names, storage class, driver, capacity, phase
+  * Quotas - name, spec, selectors(labels, annotations)
```

Comment:
I kind of would like to use "Cluster Quotas" to clearly separate them from project (namespaces) quotas. What do you think?